docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -998,6 +998,8 @@ const classes = makeStyles(theme => ({
   +     getInputRef={ref}
   ```
 
+- Rename `marginDense` and `inputMarginDense` classes to `sizeSmall` and `inputSizeSmall` to match the prop.
```

Comment:
Should we also list the change:

```diff
-<Input margin="dense" />
+<Input size="small" />

```

Reply:
I can add this change if needed

Reply:
@oliviertassinari thoughts?

Reply:
Sounds good. I forgot that `<Input margin="dense" />` was possible.

Reply:
ok, I'll add 
```
-<Input margin="dense" />
+<Input size="small" />
```
to the migration-v4.md
there is another change to add or make ?

Reply:
That’s all, thanks for addressing final comments 👍